src/equations/compressible_euler_potential_temperature_2d.jl

```diff
@@ -0,0 +1,301 @@
+using Trixi
+using Trixi: ln_mean, stolarsky_mean
+import Trixi: varnames, cons2cons, cons2prim, cons2entropy, entropy
+
+@muladd begin
+#! format: noindent
+struct CompressibleEulerPotentialTemperatureEquations2D{RealT <: Real} <:
+       AbstractCompressibleEulerPotentialTemperatureEquations{2, 4}
+    p_0::RealT
+    c_p::RealT
+    c_v::RealT
+    g::RealT
+    R::RealT
+    gamma::RealT
+    a::RealT
```

Comment:
Are all fields used in the implementation, e.g., `a`? Or is it just used by the LMARS flux and repeated there?